lib/bundler/cli/change.rb

```diff
@@ -14,6 +14,8 @@ def run
 
       raise InvalidOption, "`#{@gem_name}` could not be found in the Gemfile." unless dep
 
+      check_for_unsupported_options(dep)
```

Comment:
I am in process of writing for this function. Still to figure out how to skip before hook for one test.